doc/source/user_guide/plotting_with_bokeh.md

```diff
@@ -0,0 +1,135 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+(plotting_with_bokeh)=
+
+# Plotting with Bokeh
+
+Arviz has the option to use {doc}`Bokeh <bokeh:index>` as backend which offers interactive plots. Although most of the functions in the {ref}`plot <plot_api>` module work seamlessly with any backend, some advanced plots may require the use of backend specific features. In this guide, advanced plotting with Bokeh will be covered.
+
+This page can be downloaded as a {jupyter-download:script}`Python script <plotting_with_bokeh>`
+or as a {jupyter-download:nb}`Jupyter notebook <plotting_with_bokeh>`.
+
+```{code-cell} ipython3
+import arviz as az
+import matplotlib.pyplot as plt
+import numpy as np
+
+az.style.use("arviz-darkgrid")
+
+# Confgure Bokeh as backend
+az.rcParams["plot.backend"] = "bokeh"
+az.output_notebook()
+```
+
+:::{note}
+To display the plots as output in a Jupyter notebook, it's required to call the function  {func}`arviz.output_notebook` beforehand.
+:::
+
+:::{warning} Page in construction
+:::
+
+## Customizing plots
+
+
+### Using `backend_kwargs`
+
+The `backend_kwargs` argument can be very useful for some specific configuration. That is parameters available in {class}`bokeh:bokeh.plotting.figure.Figure`.
+
+As an example, the following code changes the color background and the plot width.
+
+```{code-cell} ipython3
+# load data
+data = az.load_arviz_data('radon')
+```
+
+```{code-cell} ipython3
+az.plot_posterior(
+    data,
+    var_names=["g"],
+    backend_kwargs={"width": 350, 
+                    "background_fill_color": "#d3d0e3"});
+```
+
+### The parameter `show`
+
+The parameter `show` is used to control whether the plot is displayed or not. This behavior can be useful when a plot will be displayed in a grid as the example in the following section.
+
+
+### Defining custom axes
+
+The `ax` argument of any `plot` function allows to use created axes manually. In the example, this parameter allows to arrange 2 different plots in a grid, set limit to the x axis and share the axes' ranges between the two plots.
+
+
+```{code-cell} ipython3
+from bokeh.io import show
+from bokeh.layouts import gridplot, row
+from bokeh.plotting import figure
+
+# load data
+observed_data = data.observed_data.y.to_numpy()
```

Comment:
```suggestion
observed_data = data.observed_data.y.values
```

not completely sure what `to_numpy()` method does, but afaik, the recommended ways to access the underlying arrays in xarray.DataArray is `.values` if we want the data as a numpy array or `.data` if we want the actual array structure used by xarray to store the data in that case (defaults to numpy but can already be dask or pydata/sparse and they are working to extend it to arbitrary array libraries)

Reply:
`to_numpy()` was replaced by `values`.